style.css

```diff
@@ -557,6 +672,95 @@ footer .hr {
     display: none;
   }
 
+  button:hover {
+    cursor: pointer;
+    color: #008552;
+    background-color: #ebf0ee;
+    transition: 0.4s;
+    border: 1px solid #008552;
+  }
+
+  button:active {
+    background-color: #008552;
+  }
+
+  .social-media-dsk img:hover {
+    transform: rotate(-20deg);
+    transition: 0.5s;
+  }
+
+  /* popup-window */
+  #popup-window {
```

Comment:
- [ ] The popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A2702)(e.g., margins):
- Please get rid of the horizontal scrolling bar and make your popup looks like the image below. 
-
![p](https://user-images.githubusercontent.com/63972670/168161400-705b5b5c-a196-4076-9147-daa2a1e74407.PNG)

Reply:
@OLIPLICHE
Please, can you elaborate more, because the windows screen cannot contain the popup height without scrolling effect.